tests/basic.bats

```diff
@@ -3,42 +3,43 @@
 load helpers
 
 @test "from" {
-  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json --image alpine)
+  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
   buildah delete $cid
   cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json         alpine)
   buildah delete $cid
-  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json         alpine  i-love-naming-things)
+  cid=$(buildah from alpine --pull --signature-policy ${TESTSDIR}/policy.json --name i-love-naming-things)
```

Comment:
It seems to work.  Only problem I have found that does not work is

OPTIONS ARGS OPTIONS

If the options are all after or before the args CLI seems to work